src/index.html

```diff
@@ -42,8 +42,8 @@
         border-color: #f1f1f1;
       }
       /* Couple of style tweaks for the report to deal with offset from the navbar */
-      a.anchor {
-        top: calc(-72.38px - 1.5em) !important;
+      .anchor {
+        padding-top: calc(72.38px + 1em) !important;
```

Comment:
why?

Reply:
When adding the new linking feature to cucumber-forge-report-generator, I ended up tweaking the css around the anchor links.  This css is used when the sidebar buttons are clicked and the `window.scrollIntoView` functionality gets executed to scroll the page to the anchor.  Since cucumber-forge-desktop has that 72.38px navbar at the top, this css tweak is needed to ensure that when the page is scrolled the desired scenario is not still under the navbar.